libdnf.spec

```diff
@@ -164,21 +177,26 @@ ERROR
         exit 1
 fi
 
+%if %{with python2}
 pushd build-py2
   make ARGS="-V" test
 popd
+%endif # with python2
 %if %{with python3}
 # Run just the Python tests, not all of them, since
 # we have coverage of the core from the first build
 pushd build-py3/python/hawkey/tests
+  sed -i '1s=^#!/usr/bin/\(python\|env python\)[0-9.]*=#!%{__python3}=' tests/run_nosetests
```

Comment:
this should be fixed properly in file rather than this hack in spec file

Reply:
Is this sed hack even needed? cmake itself should replace #!${PYTHON_EXECUTABLE} in python/hawkey/tests/tests/run_nosetests.in with correct python3 path.